pattern_import_export/models/ir_exports.py

```diff
@@ -232,19 +232,20 @@ def _generate_import_with_pattern_job(self, patterned_import):
         try:
             attachment_data = base64.b64decode(patterned_import.datas.decode("utf-8"))
             datas = self._read_import_data(attachment_data)
+            res = (
+                self.with_context(
+                    load_format="flatty",
```

Comment:
can you rebase your PR and adapt the code (flatty key is now removed)

https://github.com/akretion/pattern-import-export/pull/17/files#diff-b0c82f1e07e5526647175a9398c7a32dR244

Reply:
done